do/cloud.go

```diff
@@ -30,7 +30,7 @@ import (
 	"k8s.io/kubernetes/pkg/controller"
 )
 
-const DOAccessTokenEnv string = "DO_ACCESS_TOKEN"
+const doAccessTokenEnv string = "DO_ACCESS_TOKEN"
```

Comment:
Did `golint` complain about this? Acronyms shouldn't be caps?

Reply:
There was no need to export it, so I changed it to be unexported...